docs/changelog/3121.misc.rst

```diff
@@ -0,0 +1 @@
+Add ``/tests/demo_pkg_inline/.tox`` to ``.gitignore`` - by :user:`posita`.
```

Comment:
I'm not sure this is helpful in the context of this PR, since it's a development environment meta issue. I'm happy to remove this, if desired.